src/components/TrialBanner.js

```diff
@@ -66,7 +64,7 @@ export const TrialBanner = Utils.connectAtom(authStore, 'authState')(class Trial
     const { trialState } = profile
     const removeBanner = localStorage.getItem('removeBanner')
     if (!trialState || !isSignedIn || !acceptedTos || trialState === 'Finalized' || removeBanner === 'true') return null
-    const { [trialState]: { title, message, enabledLink, button, isWarning } } = getMessages()
+    const { [trialState]: { title, message, enabledLink, button, isWarning } } = messages
```

Comment:
I'm not sure if this matters for getAppName/trial banner, but this will result in a behavior change. The interpolation will only get executed once, if the app name changes it will not be reflected in the string.

Reply:
Yup, and that's acceptable, we expect to have to refresh after changing the app name, since the only time it can happen after first mount is config overrides.